tools/loader/ImageClassifier.cpp

```diff
@@ -99,9 +106,19 @@ void loadImagesAndPreprocess(const llvm::cl::list<std::string> &filenames,
   std::tie(imgHeight, imgWidth, isGray) = getPngInfo(filenames[0].c_str());
   const size_t numChannels = isGray ? 1 : 3;
 
+  if(imageformat != "NHWC") {
```

Comment:
It's more readable to write: `if (imageformat == "NCHW")`

Reply:
Will do